pip_deps/requirements.txt

```diff
@@ -5,3 +5,4 @@ pygobject==3.28.2
 pyusb==1.0.2
 Adafruit_BBIO==1.1.1
 smbus-cffi==0.5.1
+numpy==1.17.0
```

Comment:
Current NumPy version on Jetson is 1.16.2, should we try to keep them the same? @raytitan